ansible/all-in-one/roles/xsnippet/templates/xsnippet.yml.j2

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: xsnippet
+spec:
+  containers:
+    - name: db
+      image: "{{ xsnippet_db_image }}"
+      env:
+        - name: POSTGRES_DB
+          value: xsnippet
+        - name: POSTGRES_USER
+          value: xsnippet
+        - name: POSTGRES_PASSWORD
+          value: xsnippet
```

Comment:
nit: I'd move username/password to secrets but I'm ok to do this separately. Regardless, can you please move definitions of these credentials to Ansible's vars file?

Reply:
Done (the latter)